dbt/adapters/athena/impl.py

```diff
@@ -636,35 +636,64 @@ def persist_docs_to_glue(
         model: Dict[str, Any],
```

Comment:
it seems `model` type should also be fixed to ParsedNode which can be found here:
```
  from dbt.contracts.graph.nodes import ParsedNode
```

Reply:
Unfortunately it does not work 😔:

```
    clean_table_description = clean_sql_comment(model.description)
AttributeError: 'dict' object has no attribute 'description'
```

model is still a dict